README.md

```diff
@@ -6,11 +6,11 @@ The API is a Spring based application to suit the needs of a LAN-party.
 
 ### Tools
 -   [PostgresQL](https://www.postgresql.org/)
--   [Mailcatcher](https://mailcatcher.me/), mailcatcher creates a mailserver locally on your pc. All mail sent from the API is cought here, you end up with a mailbox with every outgoing mailaddress. Unix-like systems: `gem install mailcatcher`. Windows users can try mailcatcher as well, but [Papercut](https://github.com/changemakerstudios/papercut) has an easier installation. 
+-   [Mailcatcher](https://mailcatcher.me/), creates a mailserver locally on your pc. Mailcatcher catches all mail sent from the API and displays them in a web interface. Unix-like systems: `gem install mailcatcher`. Windows users can try mailcatcher as well, but [Papercut](https://github.com/changemakerstudios/papercut) has an easier installation. 
```

Comment:
Let's do `Mailcatcher. Creates a ....`

Reply:
If we change it that way I would say the sentence on Docker below it should be changed in the same way to be consistent.

Reply:
Let's change them all, it's a better format indeed.